src/test/java/com/adyen/PayoutTest.java

```diff
@@ -0,0 +1,103 @@
+import static com.adyen.constants.ApiConstants.AdditionalData.FRAUD_RESULT_TYPE;
+import static org.junit.Assert.assertEquals;
+
+public class PayoutTest extends BaseTest {
+
+    @Test
+    public void testStoreDetailAndSubmitThirdPartySuccess() throws Exception {
+        Client client = createMockClientFromFile("mocks/payout/storeDetailAndSubmitThirdParty-success.json");
+        Payout payout = new Payout(client);
+
+        StoreDetailAndSubmitRequest request = new StoreDetailAndSubmitRequest();
+        StoreDetailAndSubmitResponse result = payout.storeDetailAndSubmitThirdParty(request);
+
+        assertEquals("[payout-submit-received]", result.getResultCode());
+        assertEquals("8515131751004933", result.getPspReference());
+        assertEquals("GREEN", result.getAdditionalData().get(FRAUD_RESULT_TYPE));
+        assertEquals("false", result.getAdditionalData().get(FRAUD_MANUAL_REVIEW));
+    }
+
+    @Test
+    public void testStoreDetailSuccess() throws Exception {
+        Client client = createMockClientFromFile("mocks/payout/storeDetail-success.json");
+        Payout payout = new Payout(client);
+
+        StoreDetailRequest request = new StoreDetailRequest();
+        StoreDetailResponse result = payout.storeDetail(request);
+
+        assertEquals("Success", result.getResultCode());
+        assertEquals("8515136787207087", result.getPspReference());
+        assertEquals("8415088571022720", result.getRecurringDetailReference());
+    }
+
+    @Test
+    public void testConfirmThirdPartySuccess() throws Exception {
+        Client client = createMockClientFromFile("mocks/payout/modifyResponse-success.json");
+        Payout payout = new Payout(client);
+
+        ModifyRequest request = new ModifyRequest();
+        ModifyResponse result = payout.confirmThirdParty(request);
+
+        assertEquals("[payout-confirm-received]", result.getResponse());
+        assertEquals("8815131762537886", result.getPspReference());
+    }
+
+    @Test
+    public void testSubmitThirdPartySuccess() throws Exception {
+        Client client = createMockClientFromFile("mocks/payout/submitResponse-success.json");
+        Payout payout = new Payout(client);
+
+        SubmitRequest request = new SubmitRequest();
+        SubmitResponse result = payout.submitThirdparty(request);
+
+        assertEquals("[payout-submit-received]", result.getResultCode());
+        assertEquals("8815131768219992", result.getPspReference());
+        assertEquals("GREEN", result.getAdditionalData().get(FRAUD_RESULT_TYPE));
+        assertEquals("false", result.getAdditionalData().get(FRAUD_MANUAL_REVIEW));
+    }
+
+    @Test
+    public void testDeclineThirdPartySuccess() throws Exception {
+        Client client = createMockClientFromFile("mocks/payout/modifyResponse-success.json");
+        Payout payout = new Payout(client);
+
+        ModifyRequest request = new ModifyRequest();
```

Comment:
Can you please do the same as ConfirmThirdPartyRequest with DeclineThirdPartyRequest?